### DISCUSSION TITLE
Cannot create classrooms

### DISCUSSION THREAD

**Discussion by pedroprieto** [community/community#58087] [GitHub Education] [answered]
### Hi everyone,

I can't create new classrooms. It seems the slug generator is not working:
![imagen](https://github.com/community/community/assets/7632447/0c875aa0-27b9-4e5c-a03f-b3304dcd0ae6)

Any thoughts?

**Comment by LoganWS** [comment]
I'm also getting this message for creating a new assignment in a Github classroom that already exists.

**Reply by LoganWS** [reply]
<img width="1015" alt="slugcantbeblank" src="https://github.com/community/community/assets/34195798/f3cb451c-affd-4e33-be42-63773fe51423">

**Comment by pedroprieto** [comment, marked as answer]
Ok, it seems fixed now. Pretty fast!! :rocket: Thanks!! :clap:
